panda_robot_server/src/panda_robot_server/ros_bridge.py

```diff
@@ -35,14 +42,19 @@ def __init__(self, real_robot=False):
         
         # Robot control
         self.arm_cmd_pub = rospy.Publisher('env_arm_command', JointTrajectory, queue_size=1) # joint_trajectory_command_handler publisher
-        self.sleep_time = (1.0 / rospy.get_param('~action_cycle_rate')) - 0.01
+        self.sleep_time = (1.0 / rospy.get_param('~action_cycle_rate')) # - 0.01
         self.control_period = rospy.Duration.from_sec(self.sleep_time)
         self.max_velocity_scale_factor = float(rospy.get_param("~max_velocity_scale_factor"))
         self.min_traj_duration = 0.5 # minimum trajectory duration (s)
         self.joint_velocity_limits = self._get_joint_velocity_limits()
-        
+        self.velocity_limits = [2.175, 2.175, 2.175, 2.175, 2.61, 2.61, 2.61] # Farhang
+        self.zero_vel= [0., 0., 0., 0., 0., 0., 0.]  # Farhang
+        self.zero_vel_cmd = self._transform_panda_list_to_dict(self.zero_vel[0:7])  # Farhang
+        self.velocity_limits = [x * self.max_velocity_scale_factor for x in self.velocity_limits] # Farhang
+        self.robot_moving = False # Farhang
         # Robot frames
         self.reference_frame = rospy.get_param('~reference_frame', 'base')
+        #print(self.reference_frame)
```

Comment:
I guess this was forgotten and can be removed right?

Reply:
done.